mesonbuild/interpreter/interpreter.py

```diff
@@ -1886,7 +1887,7 @@ def _validate_custom_target_outputs(has_multi_in: bool, outputs: T.Iterable[str]
         CT_INSTALL_DIR_KW,
         CT_INSTALL_TAG_KW,
         MULTI_OUTPUT_KW,
-        DEPENDS_KW,
+        DEPENDS_WITH_FILES_KW,
```

Comment:
I don't really see a need to rename it.

Reply:
Explained in the description:

> Note that it does not make sense to depend on files in run_target() and test() because those only have order dependencies and doesn't need rebuild dependencies, that's why we now have DEPENDS_KW and DEPENDS_WITH_FILES_KW.

Reply:
Ah very smart. Thanks!